config/secrets.yml

```diff
@@ -72,6 +72,12 @@ default: &default
   uphold_login_client_secret: <%= ENV["UPHOLD_LOGIN_CLIENT_SECRET"] %>
   uphold_api_uri: <%= ENV["UPHOLD_API_URI"] %>
   uphold_scope: <%= ENV["UPHOLD_SCOPE"] %>
+
+  # stripe
+  stripe_publishable_key: <%= ENV["STRIPE_PUBLISHABLE_KEY"] %>
+  stripe_secret_key: <%= ENV["STRIPE_SECRET_KEY"] %>
+  stripe_client_id: <%= ENV["STRIPE_CLIENT_ID"] %>
```

Comment:
I think gemini env vars will go here too

Reply:
Actually decided to move these over to 
(services.yml)[https://github.com/brave-intl/publishers/blob/e9b7b54766e720bd96e20bb7bab293b6e8cdd8a1/config/services.yml] so that external services would be configured in one location